src/src/style.css

```diff
@@ -0,0 +1,79 @@
+:root {
+  --default-color-link: #000;
+  --color-active-link: #00acdc;
+  --background-header: #fff;
+  --box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
It's a good practice to use fallback fonts. In case the 'Roboto' font fails to load for some reason, the browser will use the generic 'sans-serif' font. But it would be better to specify one more specific fallback font before the generic one. For example, 'Arial' or 'Helvetica'. So the line of code could look like this: `font-family: Roboto, Arial, sans-serif;`